Makefile

```diff
@@ -67,39 +88,35 @@ upgrade: ## update the requirements/*.txt files with the latest packages satisfy
 		pip-compile --upgrade -o requirements/test-$$platform.txt requirements/base.in requirements/test-$$platform.in requirements/test.in ; \
 	done
 
-pull_translations: ## pull translations from Transifex
-	tx pull -a
-
-push_translations: ## push source translation files (.po) from Transifex
-	tx push -s
-
-quality: ## check coding style with pycodestyle and pylint
-	tox -e quality
+requirements.js: ## install JS requirements for local development
+	npm install
 
-jasmine: ## run Javascript tests
-	tox -e jasmine
+requirements: requirements.js ## install development environment requirements
+	pip install -qr requirements/dev.txt --exists-action w
+	pip-sync requirements/base.txt requirements/dev.txt requirements/private.* requirements/test.txt
 
 jshint: ## run Javascript linting
-	@[ -x ./node_modules/jshint/bin/jshint ] || npm install jshint
+	@[ -x ./node_modules/jshint/bin/jshint ] || npm install jshint --save-dev
 	./node_modules/jshint/bin/jshint enterprise
 	./node_modules/jshint/bin/jshint spec
 
-requirements: ## install development environment requirements
-	pip install -qr requirements/dev.txt --exists-action w
-	pip-sync requirements/base.txt requirements/dev.txt requirements/private.* requirements/test.txt
-
 test: clean ## run tests in the current virtualenv
 	pip install -qr requirements/test.txt --exists-action w
 	py.test
 
 diff_cover: test
 	diff-cover coverage.xml
 
-test-all: clean jshint quality jasmine
-	## run tests on every supported Python/Django combination
+test-all: clean jshint ## run tests on every supported Python/Django combination
```

Comment:
Should we `make static` here? Otherwise, the Jasmine tests will be using an old static build.

Reply:
Yes, good idea :+1:

Reply:
Actually I realized that at least for now, this doesn't matter, but will in the future. Right now we're just using Webpack for SASS/SCSS -> CSS.